config.json

```diff
@@ -517,6 +523,12 @@
             "ContainerName": "default"
           }
         },
+        "nodeReuse":{
```

Comment:
If it is because this option only works on windows then I would suggest considering having an MsBuildParametersWin that all the windows alias' use.